elide-model-config/src/main/java/com/yahoo/elide/modelconfig/model/Table.java

```diff
@@ -57,6 +57,8 @@
 @NoArgsConstructor
 @Builder
 public class Table implements Named {
```

Comment:
Why do we need serialization for config models?  Is this for the ConfigDataStore tests?

Reply:
Table config Model is an attribute of dynamic.TableType as shown here https://github.com/yahoo/elide/blob/9af2c6aac7f1f48dd9429421c547677b1d168bfb/elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/dynamic/TableType.java#L81. So in order to have TableType serializable had to make all it attribute classes serializable.